src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -770,8 +785,16 @@ export default {
 
             return this.productRepository
                 .get(this.product.parentId, Shopware.Context.api, this.productCriteria)
-                .then((res) => {
-                    Shopware.Store.get('swProductDetail').parentProduct = res;
+                .then(async (parent) => {
+                    if (parent.propertyIds?.length > 0) {
+                        const propertyCriteria = new Criteria(1, null);
+                        propertyCriteria.setIds(parent.propertyIds);
+                        propertyCriteria.addSorting(Criteria.sort('name', 'ASC', true));
+
+                        product.properties = await this.propertyRepository.search(propertyCriteria);
```

Comment:
'product' is not defined

Reply:
please apply the same for parent or you can extract it into a method to reuse

```js
                        const propertyCriteria = new Criteria(1, null);
                        propertyCriteria.addSorting(Criteria.sort('name', 'ASC', true));
                        propertyCriteria.setIds(product.propertyIds);

                        const result = await this.propertyRepository.search(propertyCriteria);
                        result.source = product.properties.source;

                        product._origin.properties = cloneDeep(result);
                        product.properties = result;
```

Reply:
ah, my bad 🙈